src/main.js

```diff
@@ -1,101 +1,96 @@
 import './index.css'
+import './updated-alignment-position-fields'
 import * as THREE from 'three';
 import {OrbitControls} from 'three/addons/controls/OrbitControls.js';
 import WebGL from 'three/addons/capabilities/WebGL.js';
-import {CSS3DRenderer, CSS3DObject} from 'three/addons/renderers/CSS3DRenderer.js';
-import {CSS2DRenderer, CSS2DObject} from 'three/addons/renderers/CSS2DRenderer.js';
+import {CSS2DObject, CSS2DRenderer} from 'three/addons/renderers/CSS2DRenderer.js';
 
+class Location {
+    constructor(x, y, z) {
+        this.x = x;
+        this.y = y;
+        this.z = z;
+    }
+}
+
+class Dimensions {
+    constructor(height, width, depth) {
+        this.height = height;
+        this.width = width;
+        this.depth = depth;
+    }
+}
+
+const AUDIENCE_LOCATION_Z_IF_SEATED_METERS = 1.26;
+const AUDIENCE_LOCATION_Z_IF_SEATED_OTHER = 1.623;
+const AUDIENCE_LOCATION_Z_NOT_SEATED_METERS = 4.13;
+const AUDIENCE_LOCATION_Z_NOT_SEATED_OTHER = 5.32;
+const AUDIENCE_DIMENSION_WIDTH_FACTOR = 4;
+
+// SETUP
```

Comment:
Ok, thanks! I asked AI what global variable it would suggested encapsulating. It suggested creating a MainCube class. I started going down that path, but that discovered that that later screws up the position functionality that is built into a three.js mesh. So, I guess I'll wait to see what your suggestions are on this.